lib/src/utils.dart

```diff
@@ -33,45 +35,37 @@ abstract class ValueType {
   Value toValue();
 }
 
-extension NullValueUtils on Null {
-  Value toValue() {
-    return Value(nullValue: this);
-  }
-}
-
-extension NumValueUtils on num {
-  Value toValue() {
-    return Value(numberValue: toDouble());
-  }
-}
-
-extension StringValueUtils on String {
-  Value toValue() {
-    return Value(stringValue: this);
-  }
-}
-
-extension BoolValueUtils on bool {
-  Value toValue() {
-    return Value(boolValue: this);
-  }
-}
-
+// TODO fix To toValue() for List<T?>
```

Comment:
The scope of this PR has slowly been creeping, I'd like to address making List.toValue functional in a follow up PR.